tests/integration/test_parallel_replicas_no_replicas/test.py

```diff
@@ -48,3 +48,38 @@ def test_skip_all_replicas(start_cluster, skip_unavailable_shards):
                 "skip_unavailable_shards": skip_unavailable_shards,
             },
         )
+
+@pytest.mark.parametrize("skip_unavailable_shards", [1, 0])
```

Comment:
Instead of copy-paste and adjustment of some settings, let's add the setting as parameters to the test i.e. :
```
@pytest.mark.parametrize("max_parallel_replicas", [2, 3, 100])
```
imo, there is no need to involve `parallel_replicas_min_number_of_rows_per_replica` here, - it's enough to explicitly specify `max_parallel_replicas` w/o adjusting it implicitly by `parallel_replicas_min_number_of_rows_per_replica`

Reply:
ok, Thank you for reviewing the code.